web/src/components/JSMpegPlayer.jsx

```diff
@@ -16,6 +30,8 @@ export default function JSMpegPlayer({ camera, width, height }) {
     );
 
     const fullscreen = () => {
+      requestWakeLock();
+
       if(video.els.canvas.webkitRequestFullScreen) {
         video.els.canvas.webkitRequestFullScreen();
```

Comment:
This isn't part of your PR, but fullscreen does not work on Safari (iOS or MacOS).

Reply:
It does seem like Chrome is the only browser that supports this current player. Firefox mentions the fact that requesting fullscreen is deprecated. That being said html5 video tags support fullscreen and the behavior is more consistent from a wake perspective. Something like dashplayer or similar could be used but the source video would need to be adapted for a MediaStream source.

I can give porting from JSMpegPlayer to something that uses a video element a try.